components/common/homepage/extra/statisticsBox.jsx

```diff
@@ -1,15 +1,14 @@
-import { numberWithCommas } from '../../../../utils/helper';
+import { numberWithCommas } from '../../../../utils/helper'
 
-
-const StatisticsBox= ({title, value}) => {
-   return (
-      <div className="col-md-3 video-corrected-responsive">
-         <span>
-            <big>{numberWithCommas(value)}+</big>
-            <p>{title}</p>
-         </span>
-      </div>
-   )
+const StatisticsBox = ({ title, value }) => {
+  return (
+    <div className="video-corrected-responsive">
+      <span>
+        <big>{numberWithCommas(value)}+</big>
+        <p>{title}</p>
```

Comment:
Please let's have all the boxes on the same line:
![image](https://user-images.githubusercontent.com/10332499/203933350-230857af-fdf0-4257-9d8b-a0fa66d986c4.png)

Reply:
> Please let's have all the boxes on the same line: ![image](https://user-images.githubusercontent.com/10332499/203933350-230857af-fdf0-4257-9d8b-a0fa66d986c4.png)

@Kiyoonewton please ensure that this is resolved in this branch. Because, if you don't it means you will dragging back the dev backward. Please resolve this ASAP.